db/python/tables/analysis.py

```diff
@@ -397,24 +344,24 @@ async def get_incomplete_analyses(
         """
         Gets details of analysis with status queued or in-progress
         """
+
         _query = """
 SELECT a.id as id, a.type as type, a.status as status,
-        a.output as output, a_sg.sequencing_group_id as sequencing_group_id,
+        a.output as output, JSON_ARRAYAGG(a_sg.sequencing_group_id) as _sequencing_group_ids,
```

Comment:
Can we return cohort_ids to this query, probably should've been done in the initial custom cohorts PR

Reply:
Good point, I realised this can just be updated to use the query method too, saves a bunch of duplicated code